pkgs/misc/scrcpy/default.nix

```diff
@@ -47,7 +47,6 @@ stdenv.mkDerivation rec {
     echo -n > server/meson.build
   '';
 
-  mesonFlags = [ "-Doverride_server_path=${prebuilt_server}" ];
```

Comment:
The option name seems to have always been different: https://github.com/Genymobile/scrcpy/blob/479abc8c7723d2d05b7c5027528acd91946a328c/meson_options.txt#L4
Maybe the symlink is not even necessary.

Reply:
From what I could tell the option was changed to a boolean `portable` in https://github.com/Genymobile/scrcpy/commit/eb34098add981aebd0c9aaed89ea9fa04d9ce798 but I didn't look further into it. Maybe @tomberek knows

Reply:
The symlink is still required. The `scrcpy` package works correctly with this branch's changes.